### PR TITLE
Contact branch

### PR DESCRIPTION
**Pull request by valt19**
Portfolio project by valt19.
Bug fix
Run local stylelint config command(npx stylelint "**/*.{css,scss}" --fix.
Checklist
-[ ] My code follows the style guidelines of this project

I have performed a self-review of my own code
I have commented my code, particularly in hard-to-understand areas
I have made corresponding changes to the documentation
My changes generate no new warnings

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @valt19  
image


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

- [ ] This is a pair programming activity but you have done it all alone. Please pair pragramm with your partner.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -207,5 +208,47 @@ <h4>Skills</h4>
...
+           <form action="https://formspree.io/f/xeqwozwl" method="post" >
+           
+             <input id="name" type="text" class="input"  name="user_name" placeholder="Full name" required maxlength="30">
+
+             <input id="email" type="email" class="input" name="user_email" placeholder="Email address" required>
+
+             <textarea id="message" class="msg" name="user_message" placeholder="Enter text here" required maxlength="500"></textarea>
+
+             <button type="submit" class="form-bttn" id="btnid">Get In Touch</button>
+
```
Comment (mahbubul14) [inline index.html:226]: - [ ] Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. Kindly work on it. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @valt19,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly update the getting started section. This should be an example of how you may give instructions on setting up your project locally. Also, there is no link in the author section. The text should be linkable.

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css [hunk 2]
```diff
@@ -514,8 +569,112 @@ h4 {
...
+.msg:hover {
+  transition: all 0.3s;
+  background: #dfe1e6;
+}
+
+.form-bttn:hover {
+  border: 1px solid rgb(2, 155, 35);
+  color: #6588e2;
+  background-color: #c1c7d0;
+}
```
Comment (Owusu-Desmond) [inline style.css]: - [ ] Please as requested by the previous code reviewer, interactions such as hover, active (when it's pressed), disabled and enable (default state) should be styled based on the design you selected as shown in the Figma template. Kindly work on it
  
  ### Your design
  Screenshot from 2023-02-21 16-10-19
  
  ### Figma design
  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @valt19 :wave: 

You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jasonfelice — APPROVED** [review]
Hi @valt19, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
No Linter errors✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — COMMENTED** [review]
Hi @valt19, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>

You have submitted a pair programming project without commits from your new partner(s). I can see that this might be a repeated milestone. While straightway project approval is guaranteed for repeated solo projects, this is a pair programming milestone and you're required to collaborate with your new team on the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by emmiiorji** [comment]
Hi @valt19, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>

As I mentioned earlier, you have submitted a pair programming project without commits from your new partner(s). I can see that this might be a repeated milestone. While straightway project approval is guaranteed for repeated solo projects, this is a pair programming milestone and you're required to collaborate with your new team on the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**